plugins/cpu/fu-cpu-device.c

```diff
@@ -1,9 +1,18 @@
 /*
  * Copyright (C) 2019 Mario Limonciello <mario.limonciello@dell.com>
+ * Copyright (C) 2021 Intel Corporation
  *
  * SPDX-License-Identifier: GPL-2+
  */
 
+#include <sys/utsname.h>
```

Comment:
This commit breaks Win32 CI.  Can you guard to make sure that this and the headers used for it are only for !Win32 using defines?